tests/entrypoints/openai/embedding_tests/test_encoder_decoder.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 import pytest_asyncio
 
-from ...utils import RemoteOpenAIServer
+from ....utils import RemoteOpenAIServer
```

Comment:
we can remove this test. encoder decoder models are tested in another way. Its not worth testing whether the OAI client works with these types of models